app/proguard-rules.pro

```diff
@@ -5,6 +5,7 @@
 -optimizationpasses 6
 -allowaccessmodification
 -dontpreverify
+-dontobfuscate
```

Comment:
Does it remove obfuscate or adds it?

Reply:
This flag makes R8 skip obfuscation. But it will still do the other shrink optimisations. I use it to debug crashes caused by too much shrinking, because "class x.z.y not found" is not helpful and with this flag you get the full classname again. But it shouldn't be used in production imho.

Reply:
That's what I thought. I don't know how much obfuscation matters for the size. I'm also a bit confused, you want to roll it back before the release or have another proguard file or?..